js/home.js

```diff
@@ -185,15 +186,19 @@ function init() {
     }
 
     if (!logo) {
-        // note, when adding an SVG, make sure the height/width/viewbox attributes from the top svg node are removed from the actual .svg file
-        Snap.load('./assets/hackgt.svg', function(loadedFragment) {
-            s.append(loadedFragment);
-            logo = s.select('#hackgt');
-            // useFilters && logo.attr({filter: lightShadow});
-            logo.attr({filter: logoShadow});
-            logo.attr({fill: logoColor});
-            centerLogo();
-        });
+        if (!logoDownloadTriggered) {
```

Comment:
Nit: don't really need to change but I guess this could just be `if (!logo && !logoDownloadTriggered)`

Reply:
I didn't want to do that because otherwise it will jump to the else block and call `centerLogo()` which fails if the logo isn't loaded I think